src/base/sequence_checker.h

```diff
@@ -1,7 +1,6 @@
 #pragma once
 
 #include <optional>
```

Comment:
Please revert this change/bring back the empty line between <optional> and logging.h includes here.

Also, squash everything into a single commit please.